src/test/scala/org/broadinstitute/dsde/rawls/webservice/WorkspaceApiServiceSpec.scala

```diff
@@ -523,4 +529,120 @@ class WorkspaceApiServiceSpec extends FlatSpec with HttpService with ScalatestRo
 
   // End ACL-restriction Tests
 
+  // Workspace Locking
+  it should "allow an owner to lock the workspace" in withEmptyWorkspaceApiServices { services =>
```

Comment:
Should it `allow the owner to lock an already locked workspace` ?  I don't know what the desired behavior is for that, but we should test it once we have that answer.